tasks.py

```diff
@@ -91,7 +91,7 @@ def docker_compose(context, command, **kwargs):
         command (str): Command string to append to the "docker-compose ..." command, such as "build", "up", etc.
         **kwargs: Passed through to the context.run() call.
     """
-    compose_command = f'docker-compose --project-name {context.nautobot.project_name} --project-directory "{context.nautobot.compose_dir}"'
+    compose_command = f'DOCKER_DEFAULT_PLATFORM=linux/amd64 docker-compose --project-name {context.nautobot.project_name} --project-directory "{context.nautobot.compose_dir}"'
```

Comment:
At a glance, it might be better to specify this as part of the `env` (line 108 below) rather than embedding the environment variable into the command string.

Reply:
Doh! Thanks for that suggestion @glennmatthews, didn't think about that. I'll switch it. Think we should allow this to be overridden as an argument like we do in buildx?

Reply:
No strong opinion on that question - I don't see a strong need for it at this time and it should be easy enough to add later if desired, but if you want to add it now it won't hurt anything either.